### PR TITLE
Add ChannelPermissionGate to navbar's add members

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add ChannelPermissionGate to navbar's add members

#### Ticket Link
none

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

File: components/navbar/navbar.jsx [hunk 1]
```diff
@@ -483,19 +483,26 @@ export default class Navbar extends React.Component {
...
-                                dialogType={ChannelInviteModal}
-                                dialogProps={{channel, currentUser: this.state.currentUser}}
-                            >
-                                <FormattedMessage
-                                    id='navbar.addMembers'
-                                    defaultMessage='Add Members'
-                                />
-                            </ToggleModalButton>
-                        </li>
+                        <ChannelPermissionGate
```
Comment (enahum) [inline components/navbar/navbar.jsx]: Wondering why this component doesn’t take care of everything like channelId, teamId, permissions, etc

Obviously non-blocking just curious

Comment (jespino) [reply components/navbar/navbar.jsx]: What do you mean?

Comment (saturninoabril) [reply components/navbar/navbar.jsx]: 👍 on that point. I think `teamId` should be handled by ChannelPermissionGate but I'll defer to permission's team on that.
@jespino any comment before I merge this PR?

Comment (saturninoabril) [reply components/navbar/navbar.jsx]: I'll merge this now.

**Pull request merged**